conda/cli/find_commands.py

```diff
@@ -67,14 +67,13 @@ def find_commands(include_others=True):
 
     res = set()
     for dir_path in dir_paths:
-        if not isdir(dir_path):
-            continue
         try:
             for entry in os.scandir(dir_path):
                 m = pat.match(entry.name)
                 if m and entry.is_file():
                     res.add(m.group(1))
-        except PermissionError:
+        except (FileNotFoundError, PermissionError):
```

Comment:
A call to `os.scandir('conda.py')` produces `NotADirectoryError` which is a subclass of `OsError`.

Reply:
I just tested this locally:

```
$ python -c "import os; os.scandir('yellow')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'yellow'
```

Reply:
Add an existing file to PATH.

Reply:
ugh